module#1/multi_project/core/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'java'
+}
+
+sourceCompatibility = 1.8
+
+group 'com.epam'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
+    compile project(':task1_1')
```

Comment:
Please replace this with regular dependency `compile 'com.example:utils:1.0-SNAPSHOT'`